### PR TITLE
add /help page

### PR DESCRIPTION
**Pull request by braxex**
## Description

This PR is filled to the brim with fun code for you to look at! Feel free to skip the following file types, as they are just adds, deletes & renames: `json`, `png`, `svg`, `xls`. The important files to review are noted below.

Changes: 
- Add /help page
    - `./components/accordions/Help.tsx` & `.test.tsx`
    - `./tests/cypress/tests/accessibility/get-help.spec.js`
- Create components for /help page
    - `./components/accordions/FaqAccordion.tsx` & `.test.tsx`
    - `./components/cards/EmailCard.tsx` & `.test.tsx`
- Create reusable accordion helper component and implement in new & existing components
    - `./components/accordions/AccordionItem.tsx` & `.test.tsx`
    - `./components/accordions/TemplateCardAccordion.tsx` & `.test.tsx`
- Create reusable card component and implement in new & existing components
    - `./components/accordions/Card.tsx` & `.test.tsx`
    - `./components/accordions/TemplateCard.tsx` & `.test.tsx`
- Tweak some high-level layouts, component base styles, and accessibility styling
    - `./styles/index.scss`
    - `./styles/theme.ts`

Satisfies [MDCT-214](https://qmacbis.atlassian.net/browse/MDCT-214)

### How to test

1. Pull and run locally
2. In browser, navigate to `/help` and stress test
3. Run unit tests
4. Run integration tests

### Changed Dependencies

No changed deps.

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research) tests
- [x] ~~I have added analytics, if necessary~~
- [x] ~~I have updated the documentation, if necessary~~

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

File: services/ui-src/src/components/accordions/TemplateCardAccordion.test.tsx [hunk 1]
```diff
@@ -5,23 +5,11 @@ import { RouterWrappedComponent } from "utils/testing/setupJest";
...
-/*
- * From Chakra UI Accordion test file
- * https://bit.ly/3MFtwXq
- */
-jest.mock("@chakra-ui/transition", () => ({
-  ...jest.requireActual("@chakra-ui/transition"),
-  Collapse: jest.fn(({ in: inProp, children }: any) => (
-    <div hidden={!inProp}>{children}</div>
-  )),
-}));
```
Comment (braxex) [inline services/ui-src/src/components/accordions/TemplateCardAccordion.test.tsx:19]: Moved this to `jestSetup` so the mock is available to any component that needs it.

Comment (gmrabian) [reply services/ui-src/src/components/accordions/TemplateCardAccordion.test.tsx:19]: good call!

**Review by juliuscaesar — APPROVED** [review]

**Review by BearHanded — APPROVED** [review]

**Review by karla-vm — APPROVED** [review]
Gorgeous.

**Review by gmrabian — APPROVED** [review]
going for 4

**Pull request merged**